tripy/tripy/frontend/tensor.py

```diff
@@ -178,11 +178,11 @@ def eval(self) -> runtime.MemRefValue:
 
         compiler = Compiler(trt_builder_opt_level=0)
         executable = compiler.compile(mlir, flat_ir=flat_ir)
-        executor = Executor(executable)
+        self.executor = Executor(executable)
```

Comment:
@pranavm-nvidia @parthchadha Allocated output pointer is owned by the `RuntimeClient` (`Executor` owns `RuntimeClient`), we need to ensure bind the lifetime of the runtime client to end-to-end Tripy execution.

This approach does not work if later try to access the memref pointer:
```
        input = tp.arange(1, 4, dtype=tp.float32)
        print(cp.from_dlpack(input))
```